src/components/Search.js

```diff
@@ -33,7 +33,7 @@ class Search extends Component {
 
   onSubmit(e) {
     this.props.onSearch(this.state.searchText);
-    e.preventDefault();
+    e.preventDefault(); // when we get the object, what happens to it?
```

Comment:
Was this a question for me? The `e.preventDefault()` is just something to prevent the default behaviour of the browser trying to submit the form. Before ajax requests, forms would submit data by make an entire page request, when ajax technology came about there were functions to intercept a form submit event and you could use `e.preventDefault()` to disable the default behaviour, which in this case is the old browser method of submitting the form involving a full request that takes away the user from the current page.